.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.0.3rc1
+current_version = 2.0.3rc2
```

Comment:
Main branch is targeted for 2.1.0, so 2.0.3rc2 is incorrect. If you want a pypi release it could be `2.1.0a0`or something